sssom/cli.py

```diff
@@ -15,11 +15,33 @@
 import logging
 from pandasql import sqldf
 
+# Click 'help' variabes
+help_input = 'Input file. For e.g.: SSSOM tsv file'
```

Comment:
if you leave the actual help text so far away from the options themselves, they can't serve as documentation for the code. If you need to reuse an option in several functions, you could always save the option into a variable.

Reply:
Thank you so much for the suggestion but I do not follow. Would it be possible to direct me to an example to do this the right way?

Reply:
@cthoyt this is a constant question mark for me in CLI development, see debates above. IDEs like pycharm wont read the click help text in any case, so we wont get the nice autocomplete help text when typing code no matter where the help text actually is. On the other hand, the CLI code (in this case click) needs to have the help so the user can get it when `--help`. Now: Is there any way to prevent this redundancy?

Reply:
A CLI should implement as little logic as possible. Offload as much as you can to a helper function

Reply:
Then the helper function can be idiomatic and pythonic, and the cli can just focus on click

Reply:
I dont understan. Consider this method:

```
@main.command()
@click.option('-i', '--input', required=True, type=click.Path(), help='The input file (URL or local) in SSSOM format.')
@click.option('-d', '--output-directory', type=click.Path(),help='The directory to which the separate mappings will be exported')
def split(input: str, output_directory: str):
    """
    Reading a single table in SSSOM format and splitting int by prefixes.
    
    :param input: The input file (URL or local) in SSSOM format.
    :param output_directory: The directory to which the separate mappings will be exported
    """
    split_file(input_path=input, output_directory=output_directory)
```

How can I avoid the redundancy of the docstring and click help text?

Reply:
The first line of the docstring is propagated to the command line help, but the rest of the docstring should be dropped.

Reply:
And tools like pycharm and others would recognise click options as docs, or is it just that because you know that you will never use the CLI command in pycharm, you can omit them here in favour of the docstrings?

Reply:
It's because you know you will never call a click function from any other python code or in PyCharm, exactly

Reply:
Thank you @cthoyt super interesting and lesson learned :)

Reply:
How about showing an example there? 

so instead of 
```
:param input: The input file (URL or local) in SSSOM format.
:param output_directory: The directory to which the separate mappings will be exported
```

Show an example of how the command is used correctly? Do you think that would be useful for users?

Reply:
I don't think the docstring of a command line is really the place where people show examples. You have manpages for most CLI-only tools, and SSSOM should also have user documentation with Sphinx. This is where usage and examples should go.

Reply:
Thank you so much @cthoyt !!